brage-import/src/main/java/no/sikt/nva/brage/migration/model/Publication.java

```diff
@@ -0,0 +1,99 @@
+package no.sikt.nva.brage.migration.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
+public class Publication {
+
+    private String journal;
+    private String issn;
+    private String isbn;
+    private String publisher;
+    private String partOfSeries;
+    private String id;
+
+    @JsonProperty("id")
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    @JsonProperty("partOfSeries")
+    public String getPartOfSeries() {
+        return partOfSeries;
+    }
+
+    public void setPartOfSeries(String partOfSeries) {
+        this.partOfSeries = partOfSeries;
+    }
+
+    @JacocoGenerated
+    @Override
+    public int hashCode() {
+        return Objects.hash(journal, issn, isbn, publisher, partOfSeries, id);
+    }
+
+    @JacocoGenerated
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Have you edited your equals/hash generation to include @JacocoGenerated when generating the method? If not, talk with Kir :)

Reply:
you mean that you can set intellij to add jacocogenerated when you autogenerate code?

Reply:
Yes :)